content/asset-tracking/index.textile

```diff
@@ -0,0 +1,157 @@
+- @farWithSubscriber@ := The resolution to select if above the "@proximityThreshold@":#default-resolution-constraints), with one or more subscribers.
+- @nearWithoutSubscriber@ := The resolution to select if below the "@proximityThreshold@":#default-resolution-constraints), with no subscribers.
+- @nearWithSubscriber@ := The resolution to select if below the "@proximityThreshold@":#default-resolution-constraints), with one or more subscribers.
+
+h2(#resolution-constraints). Resolution constraints
+
+Resolution constraints specify the thresholds and corresponding logical mappings for tracking @Resolution@s that are required by the default @ResolutionPolicy@, which can be adopted by @Publisher@ instances. While developers can define their own resolution constraints, Ably also provides default resolution constraints that can be used for common use cases.
+
+h3(#default-resolution-constraints). Default resolution constraints
+
+The Ably-provided default resolution constraints consist of:
+
+- @DefaultResolutionSet@ := The set of resolutions which must be defined in order to populate the @DefaultResolutionConstraints@.
+- @DefaultProximity@ := This is a distance to destination at which resolution of updates is increased to improve user experience.
+- @batteryLevelThreshold@ := This is a level of battery on the device that runs Publishing SDK that is considered low. When the battery level falls below this, the resolution of updates is reduced. Default value: 10%. In the range 0.0f (no battery) to 100.0f (full battery).
+- @lowBatteryMultiplier@ := The multipler to be applied to the interval when the battery level is below @batteryLevelThreshold@.
+
+The following example shows how the default resolution constraints are populated:
+
+```[android]
+// Prepare Resolution Constraints for an asset that will be used in the Resolution Policy
+val exampleConstraints = DefaultResolutionConstraints(
+    DefaultResolutionSet( // this constructor provides one Resolution for all states
+        Resolution(
+            accuracy = Accuracy.BALANCED,
+            desiredInterval = 1000L,
+            minimumDisplacement = 1.0
+        )
+    ),
+    proximityThreshold = DefaultProximity(spatial = 1.0),
+    batteryLevelThreshold = 10.0f,
+    lowBatteryMultiplier = 2.0f
+)
+```
+
+h2(#resolution-policy). Resolution policy
+
+Resolution policy defines the strategy by which the various @ResolutionRequests@ and preferences are translated by @Publisher@ instances into a target @Resolution@.
+
+As different use cases have different requirements for battery optimization, Ably Asset Tracking SDKs provide a flexible @ResolutionPolicy@ interface, that allows SDK users to provide their own implementation of resolution optimization logic. Ably also provides a "default resolution policy":#default-resolution-policy, which is suitable for common use cases.
```

Comment:
> "default resolution policy"

Should we just write "default resolution policy implementation" (without the quotemarks)?

Reply:
> Should we just write "default resolution policy implementation"

I couldn't find that text. However, there is a Textile link for default resolution policy, which uses quotes for link syntax.